data/mods/Aftershock/mobs/afs_robots.json

```diff
@@ -33,5 +33,58 @@
     "death_drops": {  },
     "death_function": [ "BROKEN" ],
     "flags": [ "SEES", "ELECTRONIC", "IMMOBILE", "NO_BREATHE" ]
+  },
+  {
+    "id": "afs_mon_sentinel_lx",
+    "type": "MONSTER",
+    "name": "Wraitheon Sentinel-lx",
+    "description": "Its exterior plates sable and gold, this luxurious variant of a Wraitheon drone was once kept as a bodyguard by societies' wealthiest.  Still and wrist sword extended, it resembles an ancient knight, standing an eternal watch.",
```

Comment:
```suggestion
"description": "Its exterior plates are sable and gold, this luxurious variant of a Wraitheon drone was once kept as a bodyguard by society's wealthiest. Still and with it's wrist sword extended, it resembles an ancient knight, standing an eternal watch.",
```

Reply:
Its valid to omit the "to be" verb in that context. I corrected the other parts